packages/admin-ui/src/Grid/Grid.tsx

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import { makeDecoratable } from "@webiny/react-composition";
+import { cva, type VariantProps } from "class-variance-authority";
+import { cn, withStaticProps } from "~/utils";
+
+const columnVariants = cva("", {
+    variants: {
+        span: {
+            auto: "col-auto",
+            1: "col-span-1",
+            2: "col-span-2",
+            3: "col-span-3",
+            4: "col-span-4",
+            5: "col-span-5",
+            6: "col-span-6",
+            7: "col-span-7",
+            8: "col-span-8",
+            9: "col-span-9",
+            10: "col-span-10",
+            11: "col-span-11",
+            12: "col-span-12"
+        },
+        offset: {
+            1: "col-start-2",
+            2: "col-start-3",
+            3: "col-start-4",
+            4: "col-start-5",
+            5: "col-start-6",
+            6: "col-start-7",
+            7: "col-start-8",
+            8: "col-start-9",
+            9: "col-start-10",
+            10: "col-start-11",
+            11: "col-start-12"
+        },
+        align: {
+            top: "self-start",
+            middle: "self-center",
+            bottom: "self-end"
+        }
+    },
+    defaultVariants: {
+        span: "auto"
+    }
+});
+
+interface ColumnProps
+    extends React.HTMLAttributes<HTMLDivElement>,
+        VariantProps<typeof columnVariants> {
+    children?: React.ReactNode;
+}
+
+const ColBase = React.forwardRef<HTMLDivElement, ColumnProps>(
+    ({ span, align, children, className, offset, ...props }, ref) => {
+        return (
+            <div
+                {...props}
+                className={cn(columnVariants({ span, offset, align, className }))}
+                ref={ref}
+            >
+                {children}
+            </div>
+        );
+    }
+);
+
+ColBase.displayName = "Col";
+
+const Col = makeDecoratable("Col", ColBase);
+
+const gridVariants = cva("grid", {
+    variants: {
+        gap: {
+            comfortable: "px-xl gap-lg",
+            spacious: "px-xxl gap-xl"
```

Comment:
I would not manage the outer `<Grid />` spacing with padding; it should extend across the full width of its container.

Therefore, I believe we should create a `<Container />` component or something similar that manages the spacing and positioning of its children within the viewport.

Its usage would be the following:

```jsx
<Container> // it contains, pads and optionally centers the content.
  <Grid> // it contains colums.
    <Column>Column 1</Column>
    <Column>Column 2</Column>
  </Grid>
</Container>
```